python-bindings/overview_article/tasks.py

```diff
@@ -66,7 +113,11 @@ def build_cffi(c):
 def test_cffi(c):
     """ Run the script to test CFFI """
     print_banner("Testing CFFI Module")
-    invoke.run("python3 cffi_test.py", pty=True)
+    # pty and python3 didn't work for me (win).
+    if on_win:
+        invoke.run("python cffi_test.py")
+    else:
+        invoke.run("python3 cffi_test.py", pty=True)
```

Comment:
I think this could be 
```
invoke.run("python cffi_test.py", pty=not on_win)
```
straight up `python` will work on the other systems.  I was just being pedantic. :)